File: ios/ExpoAliyunOSSModuleTypes.swift
//
//  ExpoAliyunOSSModuleTypes.swift
//  ExpoAliyunOSS
//
//  Created by Aron on 2025/9/9.
//

import ExpoModulesCore

struct CreateBucketOptions: Record {
    @Field
    var bucketName = ""
    @Field
    var permission: String?
    @Field
    var storageClass: String?
}

File: android/src/main/java/expo/modules/aliyunoss/ExpoAliyunOSSModuleTypes.kt
package expo.modules.aliyunoss

import expo.modules.kotlin.records.Field
import expo.modules.kotlin.records.Record

class CreateBucketOptions: Record {
 @Field
 var bucketName = ""
 @Field
 var permission: String? = null
 @Field
 var storageClass: String? = null
}